.idea/codeStyles/Project.xml

```diff
@@ -1,16 +1,14 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <JavaCodeStyleSettings>
```

Comment:
Why remove these? We want to avoid wildcard imports per the style guide. Or were these not doing anything?

Reply:
@BenHenning Previously we where not using this. i.e default value is sufficient i.e. 5... If you insist we can use this JavaCodeStyleSettings customisations

Reply:
My only concern about using default value is that it may change for different android studio versions

Reply:
I agree with Vinita--is it possible for us to specify a default? In general, we never want the automatic wildcard support. And no, we shouldn't be using wildcard imports ever per https://developer.android.com/kotlin/style-guide#import_statements.

Reply:
Okay will do that